src/js/modules/supervisors/FileManager.ts

```diff
@@ -256,6 +268,10 @@ class FileManager {
     if (coprocessors.length == 0) {
       return Promise.resolve();
     } else {
+      logger.info(
+        "Initiating RPC call to redpandas coprocessor service at endpoint - enable_coprocessors with data: %s",
+        coprocessors
```

Comment:
does javascript have the concept of `f-strings` in python? so you can do in-line formatting.

Reply:
@andresaristizabal would you know? I'll do some googling

Reply:
JS doesn't have a function like `"Hello, %s." % name` how python does, but in JS we can use `Hello, ${name}` this is a string templates